hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/fsck/TestContainerHealthTask.java

```diff
@@ -137,7 +140,7 @@ public void testRun() throws Exception {
         .thenReturn(Collections.emptySet());
 
     List<UnhealthyContainers> all = unHealthyContainersTableHandle.findAll();
-    assertThat(all).isEmpty();
+    assertTrue(all.isEmpty());
```

Comment:
Please keep `assertThat`, added by HDDS-10034.

```suggestion
    assertThat(all).isEmpty();
```